django_large_image/templates/django_large_image/_include/colors.html

```diff
@@ -54,16 +129,27 @@
             </div>
 
             <div id='colorsSubGroup' style="display: none;">
-              <label for="colors">Choose a colormap:</label>
-              <select id="colors" onChange='changeColors();'>
-                <option>-- none --</option>
-              </select>
-              <br />
-              <input id="mincolor" type="number" step="0.01" onChange='changeColors();'>
-              <input id="maxcolor" type="number" step="0.01" onChange='changeColors();'>
+              <div class="flex flex-col gap-2">
+                <div class="field">
+                  <label for="colors" class="pb-1 block">Choose a colormap:</label>
+                  <select
+                    id="colors"
+                    onChange='changeColors();'
+                    class="select select-sm w-full"
+                  >
+                    <option>-- none --</option>
+                  </select>
+                </div>
+                <div class="field">
+                  <input class="input input-sm w-full" id="mincolor" type="number" step="0.01" onChange='changeColors();'>
+                </div>
+                <div class="field">
+                  <input class="input input-sm w-full" id="maxcolor" type="number" step="0.01" onChange='changeColors();'>
+                </div>
+              </div>
             </div>
```

Comment:
This will need to be dealt with as well. It's currently duplicated in the configuration box that shows up when you click a band. I deleted it, but it caused the thumbnail to stop working and I couldn't figure out why.